ChatTTS/model/speaker.py

```diff
@@ -115,7 +120,7 @@ def decode_prompt(prompt: str) -> torch.Tensor:
         return torch.from_numpy(p.astype(np.int32)).view(*shp)
 
     @torch.no_grad()
-    def _sample_random(self) -> torch.Tensor:
+    def sample_random_tensor(self) -> torch.Tensor:
```

Comment:
I agree with your changes but as for this name, it'd be better to leave it unchanged.

Reply:
According to [PEP 8](https://peps.python.org/pep-0008/#method-names-and-instance-variables), 
> Use one leading underscore only for non-public methods and instance variables.
In this case, we need to call this method out of class. Therefore, it's better not to start with a leading underscore.

Reply:
I know your meaning but we want to mark this method as unrecommended. Peopole should always use the str version except for some special reasons.

Reply:
Yes, that makes sense if there is no plan to add fine-tuning script.
I'm currently working on my [fine-tuning branch](https://github.com/ain-soph/ChatTTS/tree/dev) and it seems to be working to some extent. Will the main repo maintainer consider a PR for supporting fine-tuning?

Reply:
It sounds great, we will be happy to accept your conteibution. As for the fine-tuning, since we will put the codes in the Chat class, it can contain such a internal api directly.

Reply:
> It sounds great, we will be happy to accept your contribution. As for the fine-tuning, since we will put the codes in the Chat class, it can contain such an internal api directly.

That's where I feel confused. If you want to call `Speaker._sample_random()` method in `Chat` class, then it's an out-of-class call and it means this method is public. If that's the case, it's better to not start with leading underscore if we want to obey the PEP 8 naming standard.

Reply:
I see cases for non-class methods ([example](https://github.com/pytorch/vision/blob/f5d9ecd5024d1c4f21a5620cf2cca6d07eabd469/torchvision/prototype/datasets/utils/_resource.py#L21)), but I don't know if it's still the case for class methods.

Reply:
Well, in this case, as our main purpose is to prevent users from using this method and the Speaker class is a single instance in Chat, we can regard it as a kind of `static` or `non-class`.